Makefile

```diff
@@ -23,13 +20,9 @@ install-linter:
 	$(GOMETALINTER) --install
 lint:
 	$(GOMETALINTER)
-
-# The following targets enable local testing of goreleaser.
-build-releaser: Dockerfile.release
-	$(GORELEASER_BUILD)
-release: build-releaser .goreleaser.yml
-	$(GORELEASER) release --rm-dist --debug
-snapshot: build-releaser .goreleaser.yml
+release: .goreleaser.yml
+	$(GORELEASER) release
+snapshot: .goreleaser.yml
 	$(GORELEASER) release --rm-dist --snapshot --debug
```

Comment:
I would either remove `--rm-dist` and `--debug` from here or leave it on the `release` target as well. I think I'd prefer to at least keep `--debug` on the `release` target, since I find that output useful, even on TravisCI.

Reply:
Good point ...